packages/cli-kit/src/private/node/conf-store.test.ts

```diff
@@ -52,7 +52,8 @@ describe('removeSession', () => {
 })
 
 describe('cacheRetrieveOrRepopulate', () => {
-  it('returns the cached contents when they exist', async () => {
+  // flaky test
+  it.skip('returns the cached contents when they exist', async () => {
```

Comment:
cc @isaacroldan I saw this test flake here: https://github.com/Shopify/cli/actions/runs/4263376477/jobs/7420090038

```
cli-kit: ⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯
cli-kit:  ✓ src/public/node/context/local.integration.test.ts > macAddress > returns any mac address value
cli-kit:  FAIL  src/private/node/conf-store.test.ts > cacheRetrieveOrRepopulate > returns the cached contents when they exist
cli-kit:  Test Files  1 failed | 82 passed (83)
cli-kit: AssertionError: expected 'URL2' to deeply equal 'URL1'
cli-kit:       Tests  1 failed | 486 passed (487)
cli-kit:  ❯ src/private/node/conf-store.test.ts:72:19
cli-kit:    Start at  14:41:51
cli-kit:      70|       // Then
cli-kit:    Duration  67.14s (transform 3.25s, setup 880ms, collect 29.35s, tests 33.26s)
cli-kit:      71|       // Uses the prior run to return the cached value
cli-kit:      72|       expect(got).toEqual('URL1')
cli-kit:        |                   ^
cli-kit:      73|     })
cli-kit:      74|   })
cli-kit:  ❯ runTask ../../node_modules/.pnpm/tempy@3.0.0/node_modules/tempy/index.js:18:10
cli-kit:  ❯ src/private/node/conf-store.test.ts:56:5
cli-kit:   - Expected   "URL1"
cli-kit:   + Received   "URL2"
cli-kit: ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯
```